### ISSUE TITLE
Environment not saving

### ISSUE THREAD

**Issue by mrcat282** [adam9899/Issues#91]
When I save and exit after playing, the server is stuck shutting down. I have waited for around 15 minutes but it never gets back to the start screen. So I have to open task manager and close it, but when I come back my inventory is saved but any progress I made building is gone.

Even though it says backup completed while playing, the backups in the folder are in fact from the day before so I lose my progress.

I have 8gb of RAM and had 6.2gb allocated to the game, so I tried changing the RAM allocated to around 5.5gb but this still didn't work.

Is this a RAM issue? Maybe increasing my RAM to 16gb will fix it?

**Comment by zaekeon** [comment]
This happens to us on multiplayer servers.  Everytime the server crashes (seems common), when we log back in the inventory is all the same but the world is rolled back, sometimes hours.

**Comment by TemporalWolf** [comment]
Yeah, just lost my entire minecolony to this. Tried loading a backup: most recent backup was 14 hours out of date (supposed to run every 2 hours, and I saw the messages saying it had done backups, but it appears all the backups from that session didn't save).

Not sure what the deal is, but
a) the server crash causing chunk rollbacks is real
b) the backups are not working as advertised.

**Comment by n3rdcoach** [comment]
Hi, we have (/had?) the same Problem. After the server update from 1.1 to 1.2 we were, to say it nicely, "very surprised", that the world was not saved since 14 hours. The backup worked, but was 14 hours ago too. 

I suspect the backup thread was broken somehow and didnt finish and turn on "save-on" again.

1.3 seems to use aroma backup now
